pkg/apis/work/v1alpha2/binding_types_helper.go

```diff
@@ -204,3 +204,12 @@ func (s *ResourceBindingSpec) SchedulingSuspended() bool {
 
 	return *s.Suspension.Scheduling
 }
+
+// ExplicitPriority returns the explicit priority declared
+// by '.spec.SchedulePriority.Priority'.
+func (s *ResourceBindingSpec) ExplicitPriority() int32 {
+	if s.SchedulePriority == nil {
+		return 0
+	}
+	return s.SchedulePriority.Priority
+}
```

Comment:
```suggestion
// SchedulePriorityValue returns the scheduling priority declared
// by '.spec.SchedulePriority.Priority'.
func (s *ResourceBindingSpec) SchedulePriorityValue() int32 {
	if s.SchedulePriority == nil {
		return 0
	}
	return s.SchedulePriority.Priority
}
```
Given that ResourceBinding may introduce another priority other than the schedule aspect, I'd like to give it a more straightforward name.